### PR TITLE
Extract map body from json_jq output

### PR DESCRIPTION
**Pull request by jxu-nflx**
Pull Request type
----
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [ ] Other (please describe):

**NOTE**: Please remember to run `./gradlew spotlessApply` to fix any format violations.

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
Issue #
Nested json object in taskModel output is not resolved properly [here](https://github.com/Netflix/conductor/blob/33062f3ae506e49947bd11157dce279254f00347/core/src/main/java/com/netflix/conductor/core/utils/ParametersUtils.java#L244), so extract the map body out and put that into taskModel output instead.

Alternatives considered
----

_Describe alternative implementation you have considered_

### CODE REVIEW

File: test-harness/src/test/resources/nested_json_jq_transform_integration_test.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "nested_json_jq_transform_wf",
+  "version": 1,
+  "tasks": [
+    {
+      "name": "default_variables",
+      "taskReferenceName": "default_variables",
+      "description": "Conductor does not yet support inputTemplate on a workflow (but it's coming soon!), so we default some variables",
```
Comment (apanicker-nflx) [inline test-harness/src/test/resources/nested_json_jq_transform_integration_test.json:8]: do you want to remove this description?

File: test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy [hunk 2]
```diff
@@ -148,4 +152,39 @@ class JsonJQTransformSpec extends AbstractSpecification {
             tasks[0].outputData.containsKey("result") && tasks[0].outputData.containsKey("resultList")
         }
     }
+
+    def "Test workflow with nested json jq transform task succeeds"() {
```
Comment (apanicker-nflx) [inline test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy:156]: s/nested/sequential/

File: test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy [hunk 3]
```diff
@@ -24,9 +24,13 @@ class JsonJQTransformSpec extends AbstractSpecification {
     @Shared
     def JSON_JQ_TRANSFORM_WF = 'test_json_jq_transform_wf'
 
+    @Shared
+    def NESTED_JSON_JQ_TRANSFORM_WF = 'nested_json_jq_transform_wf'
+
     def setup() {
         workflowTestUtil.registerWorkflows(
                 'simple_json_jq_transform_integration_test.json',
+                'nested_json_jq_transform_integration_test.json'
```
Comment (apanicker-nflx) [inline test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy:33]: s/nested/sequential/

File: test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy [hunk 4]
```diff
@@ -24,9 +24,13 @@ class JsonJQTransformSpec extends AbstractSpecification {
     @Shared
     def JSON_JQ_TRANSFORM_WF = 'test_json_jq_transform_wf'
 
+    @Shared
+    def NESTED_JSON_JQ_TRANSFORM_WF = 'nested_json_jq_transform_wf'
```
Comment (apanicker-nflx) [inline test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy:28]: s/nested/sequential/

**Review by apanicker-nflx — APPROVED** [review]

File: test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy (hunk 2 above)
Comment (jxu-nflx) [reply test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy:156]: do you mean to rename from nested to sequential? My intention is to test the nested json object from jq_transform output, thus the prefix of "nested" naming

Comment (apanicker-nflx) [reply test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy:156]: Yes, thats the suggestion. I understand the intention here, but the naming of the spec and file indicate that the tasks are nested, consider renaming to ```json jq transform task with nested json object``` to explicitly call this out.

**Pull request merged**